### ISSUE TITLE
is this project dead?

### ISSUE THREAD

**Issue by colevr1** [pylorak/TinyWall#43]

**Comment by pylorak** [comment]
As clearly communicated on the front README and in [this](https://www.patreon.com/posts/open-sourcing-on-79924065) post, I will not be working on new features myself. I will issue fixes for critical bugs if any pop up, and I will integrate PRs if I feel they are still in line with the original direction TinyWall was heading. If you are unsure if your PR will be accepted, you can ask about it and discuss it with me.

**Comment by pylorak** [comment]
TLDR; For multiple reasons (subpar code quality, disrespect for original developer, political activism in code), I cannot endorse that fork.

Details:

The developer submitted a pull request about a year ago, and upon inspection it turned out he introduced about a dozen bugs into TinyWall. [See my feedback here.](https://github.com/pylorak/TinyWall/pull/5#issuecomment-1496682952). I asked him to fix the mistakes, but he decided to fork the project instead of spending time to correct them. Now I checked his latest code in the fork to see if things have improved. It appears that (with the exception of a few) he still didn't fix most bugs I pointed out. The recent bug-fixes he did publish are actually for even newer bugs he introduced and aren't present in actual TinyWall or in my feedback from then.

He also has multiple commits where he is infusing TinyWall's code with open political activism (things like removing technologies, disabling localizations for a large group of people, and leaving political messages in commits). TinyWall is free of politics. meaning all users and developers are welcome no matter what your political standings are.

But maybe most importantly, he ignored my ask that if you fork TinyWall and continue developing it on your own then you do that under a different project name. This was a general request to all forkers, but is all the more important in his specific case due to his activism. He is allowed to fork the project and exclude any number of users, or use Git commits for his political messaging, but I do not appreciate when he pretends that TinyWall has anything to do with these things. When he publishes his code or binaries under the same project name and the next subsequent version number, he is misleading all users into thinking the project took a political stance. And if I linked and redirected my users there, I'd be communicating that I approve of his practice.

To sum it up, as you can imagine I am not fond of endorsing a fork that obliviously introduces bugs, actively excludes people from his userbase, misuses TinyWall's code for political messaging, and meanwhile pretends that his project is THE new TinyWall.

**Comment by ilgrank** [comment]
@pylorak : Thanks for taking your time to clarify the situation, I did not know about the past PR and I honestly did not take the time to review the code and its political comments.
Also, thanks for pointing out that new bugs had been introduced, that's one thing an 'average' user like me would never had noticed on its own.
